exonum-java-binding/core/src/main/java/com/exonum/binding/runtime/FrameworkModule.java

```diff
@@ -33,7 +32,6 @@ protected void configure() {
     bind(Server.class).toProvider(Server::create).in(Singleton.class);
     bind(ViewFactory.class).toInstance(ViewProxyFactory.getInstance());
     // todo: Consider providing an implementation of a Node —
-    // requires changing its contract.
-    bind(UserServiceAdapter.class);
```

Comment:
This guy is [JIT](https://github.com/google/guice/wiki/JustInTimeBindings)-binded